kernel/README.md

```diff
@@ -0,0 +1,85 @@
+# Linux\* container kernel
+
+This directory contains the sources to update rpm specfiles and debian source
+control files to generate ``linux-container``, the kernel needed to boot 
+ Kata Containers.
+
+With these files we generated Fedora and Ubuntu packages for this component.
+
+``./update_kernel.sh [VERSION]``
+
+
+This script will generate and update sources to create ``linux-container``
+packages.
+
+  * linux-container.spec
+  * lnux-container_X.X.XX-XX.dsc
+
+By default, the script will get the latest 4.14 longterm kernel version from
```

Comment:
This text is going to go "stale" - can we change to using `install_kata_kernel.sh` ?